hubble/relay.go

```diff
@@ -409,3 +410,24 @@ func (k *K8sHubble) createRelayClientCertificate(ctx context.Context) error {
 
 	return nil
 }
+
+func (k *K8sHubble) PortForwardCommand(ctx context.Context) error {
+	cmd := "kubectl"
+	args := []string{
+		"port-forward",
+		"-n", k.params.Namespace,
+		"svc/hubble-relay",
+		"--address", "0.0.0.0",
+		"--address", "::",
+		fmt.Sprintf("%d:80", k.params.PortForward)}
```

Comment:
note: this is assuming that the relay service was deployed without TLS enabled (server side).